### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by georgyangelov** [colinhacks/zod#606] [General]
@colinhacks and other contributors, thank you for creating and maintaining this library!

I came across this lib ~a month after it was here on Github and I instantly loved the interface - the elegant usage and the powerful functionality and TS integration. So much that I made the decision to start using it in a production project from v1, in a project that already had yup included.

It really shows that this project receives a lot of love - with all the improvements, tests, benchmarks, github actions and fixes.

**Comment by colinhacks** [comment]
Thanks Georgy! It's always great to get messages like this :) 

> It really shows that this project receives a lot of love

The community of Zod users and contributors is uncommonly smart and friendly, and I'm extremely grateful for that :) Thanks to everyone who's made it better! @o-alexandrov @scotttrinh @alii @lilred and so many more...you all rock.
